File: src/main/java/org/example/expert/domain/todo/dto/request/TodoSearchCondition.java
package org.example.expert.domain.todo.dto.request;

import lombok.AllArgsConstructor;
import lombok.Getter;

import java.time.LocalDate;

@Getter
@AllArgsConstructor
public class TodoSearchCondition {
    private final String title;
    private final LocalDate createdFrom;
    private final LocalDate createdTo;
    private final String managerNickname;
}

File: src/main/java/org/example/expert/domain/todo/dto/request/TodoSearchCondition.kt
package org.example.expert.domain.todo.dto.request

import java.time.LocalDate

class TodoSearchCondition(
    val title: String? = null,
    val createdFrom: LocalDate? = null,
    val createdTo: LocalDate? = null,
    val managerNickname: String? = null
)
